src/status_im/contexts/wallet/send/select_asset/view.cljs

```diff
@@ -93,28 +93,24 @@
         on-close       #(rf/dispatch [:navigate-back-within-stack :wallet-select-asset])]
     (fn []
       [rn/safe-area-view {:style style/container}
-       [rn/scroll-view
-        {:content-container-style      {:flex 1}
-         :keyboard-should-persist-taps :handled
-         :scroll-enabled               false}
```

Comment:
I guess we need to move the prop (`keyboard-should-persist-taps`) to the outer scroll view.

Reply:
My bad. I overlooked.